python/map_closures/pipeline.py

```diff
@@ -218,7 +221,8 @@ def _log_to_console(self):
 
         console = Console()
         table = Table(box=box.HORIZONTALS, title=f"MapClosures detected for {self._dataset_name}")
-        table.caption = f"Loop Closure Distance Threshold: {self.closure_distance_threshold} m"
+        if hasattr(self._dataset, "gt_poses") and self._eval:
```

Comment:
This is not necessary here actually, just put `table.caption = f"Detected MapClosures"` instead of the entire if block